### PR TITLE
Add tests for average when dims is missing/incorrect

### LINKED ISSUE #135: average operator and usage of original files?
## Description

In release 0.4.0 there is a quick fix:
https://github.com/roocs/rook/blob/06f255ec49138086c59f732646cd8c1e304bfad1/rook/director/director.py#L85

When using the average operator we don't return original files. Right?

## Environment

* rook version used, if any:
* Python version, if any:
* Operating System:

## Steps to Reproduce

```
Paste the command(s) you ran and the output.
```

## Additional Information

#130

### PR DESCRIPTION
**Pull request by ellesmith88**
## Overview

This PR fixes #135 

Changes:

* Added tests to look at behaviour when dims is not provided, when it is an empty string and when it is an unknown dimension

## Additional Information

Temporarily use `clisops` master until next `clisops` release
Links to other issues or sources.

### CODE REVIEW

File: rook/processes/wps_average.py [hunk 1]
```diff
@@ -22,19 +22,19 @@ def __init__(self):
...
                 data_type="string",
-                default="c3s-cmip5.output1.ICHEC.EC-EARTH.historical.day.atmos.day.r1i1p1.tas.latest",
                 min_occurs=1,
                 max_occurs=1,
             ),
             LiteralInput(
                 "dims",
                 "Dimensions",
-                abstract="Please specify dimensions for averaging.",
+                abstract="Diimensions to avergae over. " "Example: time",
```
Comment (agstephens) [inline rook/processes/wps_average.py:34]: @ellesmith88 this should probably read:
`abstract="Dimensions to average over. Example: time",

**Comment by agstephens** [comment]
@ellesmith88 some checks are failing on this PR. Not clear why from a quick look.

**Comment by ellesmith88** [comment]
> @ellesmith88 some checks are failing on this PR. Not clear why from a quick look.

@agstephens Sorry, should have said. I made the change in `clisops`
So the update in daops: https://github.com/roocs/daops/pull/65 and this PR both depends on the PR in `clisops `: https://github.com/roocs/clisops/pull/162

I've left this rook PR and the `daops` PR as drafts until the `clisops` one is merged (If that's where we want to fix the issue)

File: rook/processes/wps_average.py (hunk 1 above)
Comment (ellesmith88) [reply rook/processes/wps_average.py]: Fixed this, thank you

File: environment.yml [hunk 2]
```diff
@@ -31,3 +31,5 @@ dependencies:
 - aiohttp
 # tests
 - pytest
+- pip:
+  - clisops @ git+https://github.com/roocs/clisops.git@master#egg=clisops
```
Comment (cehbrecht) [inline environment.yml:35]: I would skip the "pip from git" here ... I would keep it only in `requirements.txt`, as it is already.

**Review by cehbrecht — APPROVED** [review]
Looks all good 👍

**Pull request merged**